server.js

```diff
@@ -13,8 +13,8 @@ const headers = {
 // Middleware
 app.use(express.static(path.join(__dirname, '..', 'client/dist')));
 app.use(cors());
-app.use(express.json());
-app.use(express.urlencoded({ extended: true }));
+app.use(express.json({ limit: '50mb' }));
```

Comment:
this is interesting, is it for the photos? or for the text entered?

Reply:
It was, per this one tutorial I was following. Might not be helpful anymore. Removed.